ar_commands.rb

```diff
@@ -5,27 +5,28 @@
 require_relative "models/painting"
 
 # 1. Create a new Artist (Vincent Van Gogh) and save him to your database
-
+Artist.create(name: "vincent Van Gough", nationality: "Dutch")
 # 2. Create a new Painting (Starry Night) and save it to your database
-
+Painting.new(title: "other starry night", artist_id: 13)
 # 3. Return all Artists
-
+Artist.all
 # 4. Return all Paintings
-
+Painting.all
 # 5. Return the artist with an id of 2
-
+Artist.find(2)
 # 6. Return the artist with a name of "Vincent Van Gogh"
-
+Artist.find_by(name: "Vincent van Gogh")
 # 7. Return all the artists from Spain
-
+Artist.find_by(nationality: "Spanish")
```

Comment:
`.find_by` only returns the first instance that matches that query, I think you want `.where` which will return *all* matches